app/registries/menu/loader.rb

```diff
@@ -98,6 +98,7 @@ def self.load
         menu.sub_menu :lab_features_menu, :caption => N_('Lab Features'), :icon => 'fa fa-flask' do
           menu.item :host_wizard,
             :caption => 'Host Wizard',
+            :url_hash => {:controller => 'api/v2/hosts', :action => 'create'},
```

Comment:
Can you add spaces pls? Idk why Rubocop is not complaining about it, but it should and probably would once it wakes up xD
```suggestion
            :url_hash => { :controller => 'api/v2/hosts', :action => 'create' },
```

Reply:
Lol, will do! My ruby environment is likely still not setup correctly.